### PR TITLE
use prettier-js as default javascript formatter

### PR DESCRIPTION
**Pull request by vmalloc**

### CODE REVIEW

**Comment by drrlvn** [comment]
@r-darwish Thoughts?

**Comment by r-darwish** [comment]
It's a good thing to have a formatter in JavaScript. I'm in favor.

File: init.el [hunk 1]
```diff
@@ -923,4 +923,8 @@ _M-p_: Unmark  _M-n_: Unmark  _q_: Quit"
   :ensure
   :bind (("M-g M-o" . dump-jump-quick-look)))
 
+(use-package prettier-js
+  :ensure
+  :init (add-hook 'js2-mode-hook #'prettier-js-mode))
```
Comment (drrlvn) [inline init.el]: Use `:hook` instead of manually calling `add-hook`. You can see examples elsewhere in `init.el`.

**Comment by vmalloc** [comment]
Fixed and pushed

**Comment by drrlvn** [comment]
Great, thanks!

**Pull request merged**